.circleci/config.yml.j2

```diff
@@ -51,7 +56,11 @@ workflows:
           extra_args:
             {#- If you modify this, make sure you also modify it in the 'nightly' workflow -#}
             {%- if not edge_build %}
+            {%- if distribution=='bullseye'%}
+            --build-arg VERSION=$(curl https://pip.astronomer.io/v2/apache-airflow/latest-{{ airflow_version_wout_dev }}.build)
+            {%- else %}
             --build-arg VERSION=$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-{{ airflow_version_wout_dev }}.build)
```

Comment:
Btw, we will want to do this for 2.1.4 also. Probably worth leaving 1.10.15 alone, only have to support it for 2 more months.

Reply:
Do you mean building 2.1.4 from airflow-private?